gamestonk_terminal/cryptocurrency/coingecko/pycoingecko_view.py

```diff
@@ -0,0 +1,1570 @@
+        description="Shows top cryptocurrency categories by market capitalization",
+    )
+
+    try:
+        ns_parser = parse_known_args_and_warn(parser, other_args)
+        if not ns_parser:
+            return
+
+        df = gecko.get_top_crypto_categories()
+        print(
+            tabulate(
+                df,
+                headers=df.columns,
+                floatfmt=".0f",
+                showindex=False,
+                tablefmt="fancy_grid",
+            )
+        )
+        print("")
+
+    except Exception as e:
+        print(e)
+        print("")
+
+
+def recently_added(other_args: List[str]):
+    """Shows recently added coins from "https://www.coingecko.com/en/coins/recently_added"
+
+    Parameters
+    ----------
+    other_args: List[str]
+        Arguments to pass to argparse
+
+    """
+    parser = argparse.ArgumentParser(
+        prog="recently_added",
+        add_help=False,
+        description="Shows top cryptocurrency categories by market capitalization",
+    )
+
+    try:
+        ns_parser = parse_known_args_and_warn(parser, other_args)
+        if not ns_parser:
+            return
+
+        df = gecko.get_recently_added_coins()
+        print(
+            tabulate(
+                df,
+                headers=df.columns,
+                floatfmt=".0f",
+                showindex=False,
+                tablefmt="fancy_grid",
+            )
+        )
+        print("")
+
+    except Exception as e:
+        print(e)
+        print("")
+
+
+def stablecoins(other_args: List[str]):
+    """Shows stablecoins data from "https://www.coingecko.com/en/stablecoins"
+
+    Parameters
+    ----------
+    other_args: List[str]
+        Arguments to pass to argparse
+
+    """
+    parser = argparse.ArgumentParser(
+        prog="stablecoins",
+        add_help=False,
+        description="Shows stablecoins by market capitalization",
+    )
+
+    try:
+        ns_parser = parse_known_args_and_warn(parser, other_args)
+        if not ns_parser:
+            return
+
+        df = gecko.get_stable_coins()
+        print(
+            tabulate(
+                df,
+                headers=df.columns,
+                floatfmt=".0f",
+                showindex=False,
+                tablefmt="fancy_grid",
+            )
+        )
+        print("")
+
+    except Exception as e:
+        print(e)
+        print("")
+
+
+def yield_farms(other_args: List[str]):
+    """Shows Top Yield Farming Pools by Value Locked from "https://www.coingecko.com/en/yield-farming"
+
+    Parameters
+    ----------
+    other_args: List[str]
+        Arguments to pass to argparse
+
+    """
+    parser = argparse.ArgumentParser(
+        prog="stablecoins",
+        add_help=False,
+        description="Shows Top Yield Farming Pools by Value Locked",
+    )
+
+    try:
+        ns_parser = parse_known_args_and_warn(parser, other_args)
+        if not ns_parser:
+            return
+
+        df = gecko.get_yield_farms()
```

Comment:
In this one the table is just too long. In my case I couldn't even scroll through the header.

An idea here would be to allow the user to specify the order of the data they want to see and the number of elements.

As default you could do something like showing first 10 and ordering by one of those last 2 columns.

Also, if all the columns contain the same word, e.g. Hourly/Yearly, it is probably worth incorporating it on the column name, so that we avoid likelihood of overrunning screen.